spec/rails_helper.rb

```diff
@@ -25,15 +28,14 @@
   config.include ResponseJSON
   config.before(:each) do
     fixture_file = File.open("#{fixture_path}/movie_titles.json").read
-    stub_request(:get, "https://api.themoviedb.org/3/discover/movie?api_key=#{Rails.application.credentials.moviedb[:api_key]}&include_adult=false&language=en-US&page=1&sort_by=popularity.desc").
-      with(
-        headers: {
-        'Accept'=>'*/*',
-        'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
-        'Host'=>'api.themoviedb.org',
-        'User-Agent'=>'rest-client/2.1.0 (linux-gnu x86_64) ruby/2.5.1p57'
-        }).
-      to_return(status: 200, body: fixture_file, headers: {})
+    stub_request(
+      :get,
+      "https://api.themoviedb.org/3/discover/movie?api_key=#{
+        Rails.application.credentials.moviedb[:api_key]
+      }&include_adult=false&language=en-US&page=1&sort_by=popularity.desc"
+    ).to_return(status: 200, body: fixture_file, headers: {})
 
   end
-end
\ No newline at end of file
+  config.before(:each) { StripeMock.start }
```

Comment:
you don't need two before(:each) blocks, you already have one for your stub and you could've added this to that one

Reply:
Noted for this. Thanks